src/components/flex/flex_item.tsx

```diff
@@ -61,18 +96,30 @@ export const OuiFlexItem: FunctionComponent<
 > = ({
   children,
   className,
-  grow = true,
+  grow = true, // default true -> flex-grow: 1 and flex-basis: 0%
+  shrink = null, // default null for flex-shrink
+  basis = null, // default null flex-basis
   component: Component = 'div',
   ...rest
 }) => {
   validateGrowValue(grow);
+  validateShrinkValue(shrink);
+  validateBasisValue(basis);
 
   const classes = classNames(
     'ouiFlexItem',
     {
       'ouiFlexItem--flexGrowZero': !grow,
```

Comment:
I think these classes are required for both `shrink` and `basis`, since the `ouiFlexItem--flexShrink[number]` classes will only be used if the value is truthy. So if not, there needs to be some class that is applied to provide the "default" styling.

Reply:
Default for `flex-basis` is `0%` always, unless you give the prop `grow=false`. This default value is defined in _flex_group.scss` ([line 27](https://github.com/opensearch-project/oui/blob/29aff44efc14701442b53b8bd5e8bc136013c0d7/src/components/flex/_flex_group.scss#L27)). For the case of `flex-shrink` the default value is given by the default in the browser, which is `flex-shrink:1`. But I can be redundant and add it anyway in `_flex_group.scss` under `.ouiFlexItem` class. That will be OK?

Reply:
Yeah, `grow` establishes that as precedent